docs/migration-apiserver-to-crds.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Migration from API server to CRDs
+layout: docwithnav
+---
+
+Service Catalog upgrade from version 0.2.x (and earlier) to 0.3.x needs a data migration. 
```

Comment:
"Upgrading Service Catalog from versions 0.2.x (and earlier) to 0.3.x requires a database migration."